demand-capacity-mgmt-backend/src/main/resources/application.yml

```diff
@@ -36,17 +47,28 @@ spring:
 
   flyway:
     enabled: true
-    url: jdbc:postgresql://${DCM_DATASOURCE_HOST:localhost:5432}/${DCM_DATASOURCE_NAME:dcm}
+    url: jdbc:postgresql://${DCM_DATASOURCE_HOST:localhost:15432}/${DCM_DATASOURCE_NAME:dcm}
     user: ${DCM_DATASOURCE_USER:dcm}
     password: ${DCM_DATASOURCE_PASS:dcm}
     schemas: migrations
-    locations: classpath:db/migration/postgresql
-    
+    locations: classpath:db/migration
+
 keycloak:
-  baseUrl: http://localhost:8888
-  realm: dcm
-  clientId: dcmauth
-  clientSecret: ${DCMSECR}
+  baseUrl: ${KEYCLOAK_BASE_URL:http://localhost:28080}
+  realm: ${KEYCLOAK_REALM_NAME:dcm}
+  clientId: ${KEYCLOAK_CLIENT_ID:dcmauth}
+  clientSecret: ${DCMSECR:test}
   grant_type_token: password
   grant_type_refresh_token : refresh_token
 
+springdoc:
+  swagger-ui:
+    oauth:
+      clientId: ${keycloak.clientId}
```

Comment:
consider adding default value if env var is not set. e.g. `${keycloak.clientId:dcmauth}`

Reply:
We have reused the variable defined in line no: 59. we already have a default value `dmauth`

Reply:
Thanks for pointing that. As config initialization did not work for me on windows, I thought this is the problem.

I consider this resolved.